examples/sdk-snapgrid/src/components/Hero/Hero.module.css

```
@@ -0,0 +1,318 @@
+  width: 500px;
+  height: 500px;
+  background: radial-gradient(
+    circle,
+    rgba(245, 240, 232, 0.03) 0%,
+    transparent 68%
+  );
+  pointer-events: none;
+  animation: ambientBreathe 14s ease-in-out infinite reverse;
+}
+
+@keyframes ambientBreathe {
```

Stylelint (17.9.0)

[error] Expected keyframe name "ambientBreathe" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)